packages/core/src/components/ScrollTo/withTooltip.tsx

```diff
@@ -8,7 +8,7 @@ import { withTooltip as withTooltipUtil } from "@core/hocs";
 const hideTooltip = (event: React.MouseEvent<HTMLDivElement>) => {
   const isOverFlow =
     (event.target as HTMLDivElement).children.length > 1
-      ? Array.of(...(event.target as HTMLDivElement).children).some(
+      ? Array.from((event.target as HTMLDivElement).children).some(
```

Comment:
Unrelated, but if we want the `HTMLDivElement`, we should be using `event.currentTarget`
```suggestion
      ? Array.from((event.currentTarget).children).some(
```